src/styles/theme.ts

```diff
@@ -41,7 +41,7 @@ export const lightTheme: Theme = {
 		page: ghostWhite,
 	},
 	text: {
-		primary: 'black',
+		primary: charcoalBlack,
```

Comment:
This will change the font color on all projects (not only memberships) from black to charcoalBlack (#212529). @andreider04 is this the expected result?

Reply:
@diegonzs-roll confirmed that we should use this color: https://tryroll.slack.com/archives/CU4PZBFHS/p1660737198647449?thread_ts=1660648761.240699&cid=CU4PZBFHS